stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListViewModelBinding.kt

```diff
@@ -62,4 +63,28 @@ public fun MessageListViewModel.bindView(view: MessageListView, lifecycleOwner:
     }
     loadMoreLiveData.observe(lifecycleOwner, view::setLoadingMore)
     targetMessage.observe(lifecycleOwner, view::scrollToMessage)
+
+    val attachmentReplyHandler = object : AttachmentGalleryActivity.AttachmentReplyOptionHandler {
+        override fun onClick(attachmentData: AttachmentGalleryActivity.AttachmentData) {
+            TODO("Not yet done!")
```

Comment:
Is this waiting for replies to be enabled?

Reply:
It's just not implemented yet. I'll be working on implementing the handler today.

Reply:
How is it going?
Is it going to be covered on this PR?
Or maybe we can start to merge it and cover it into another PR?

Reply:
Yes, let's move "reply" to the next one. Resolving conflicts to push this PR forward.